### DISCUSSION TITLE
SOLVED - Trying to solve elephant foot issue

### DISCUSSION THREAD

**Discussion by Darkknight350** [sn4k3/UVtools#339] [Q&A] [answered]
Hello There,

in this tutorial: https://github.com/sn4k3/UVtools/wiki/Calibration:-Elephant-Foot

was written the following steps to change the issues by dimming (the way by erode had i found)

_**Future slicing of files**

...

If you goes with Dimming:

Go to: Tools - Pixel arithmetic
Click "Elephant foot compensation" preset
Put the wall thickness given from the calibration part (W)
Put the brightness given from the calibration part (B)
Save a profile with printer name, resin name for latter re-use and reference
Hit apply - done_

Sorry i haven´t found Pixel arithmetic

There is pixel dimming and
Arithmetic

but none of them has a Elephant foot compensation" preset or the possibility to enter (W) and (B)

may anyone tell me where to find the right things

Thanks in advance

Darkknight350

**Comment by sn4k3** [comment, marked as answer]
What version of UVtools do you run? It is present on many past versions (Since v2.20)

![image](https://user-images.githubusercontent.com/113281/141655381-c64dba44-750c-481f-ae1c-d862c6166d5f.png)

**Comment by Darkknight350** [comment]
And here we go...

mea culpa

i was on 2.9.1 , now i´ve updated to 2.24.3 

and here ist is.

Thank you.
I expected the app updating itself, i thought it did in the past, anyway

Solved

**Reply by sn4k3** [reply]
Pior to 2.10.x, it has a bug that were unable to detect new versions with 2 digits between dots. You will had to wait till version 2.91 for it detect a new version update :P
